src/tests/JIT/Regression/JitBlue/Runtime_55129/Runtime_55129.cs

```diff
@@ -0,0 +1,101 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.CompilerServices;
+
+public class Program
```

Comment:
Nit: I think it would be good to name the test classes uniquely (e. g. `Runtime_55129`, `Runtime_55129_1`, etc) so that, when looking at the diffs / replay failures, it is easier to find the sources.